features/intl-locale.yml

```
@@ -0,0 +1,20 @@
+name: Intl.Locale
+description: "The `Intl.Locale` API identifies locales and contains locale data, such as language, region and script codes."
```

Since we do have two features here, I think the "basic" version of this is basically parsing. Or maybe "parses or constructs"? What do you think?

(I've also tried to illustrate with examples.)

```suggestion
description: "The `Intl.Locale` API parses Unicode locale identifiers, with language, region, and script codes, such as `zh-Hans-CN` or `en-GB`."
```

---

I think what you suggest is great. For some reason I thought this API identifies a locale from any string ("de-foo" falls back to "de") but reading further it wants a Unicode locale identifier and will throw otherwise.